src/abbfreeathome/devices/cover_actuator.py

```diff
@@ -41,7 +51,7 @@ def __init__(
         room_name: str | None = None,
     ) -> None:
         """Initialize the Free@Home CoverActuator class."""
-        self._state: int | None = None
+        self._state: str
```

Comment:
This internal attribute was correct, this should be `CoverActuatorState`

```suggestion
        self._state: CoverActuatorState = CoverActuatorState.unknown
```